src/general/OptionsMenu.cs

```diff
@@ -162,6 +162,11 @@ public class OptionsMenu : Control
     [Export]
     public NodePath CustomUsernamePath;
 
+    // Sound tab statics
+    private List<string> languages = TranslationServer.GetLoadedLocales().Cast<string>()
+        .OrderBy(i => i, StringComparer.InvariantCulture)
+        .ToList();
```

Comment:
This needs to be static to actually cache things...
For that you need to take a look at how the game version is loaded in Constants.cs to see how the lazy load is implemented there.